numpyro/svi.py

```diff
@@ -70,12 +70,10 @@ def init_fn(rng, model_args=(), guide_args=(), params=None):
                 constraint = site['kwargs'].pop('constraint', constraints.real)
                 transform = biject_to(constraint)
                 if isinstance(transform, ComposeTransform):
-                    base_transform = transform.parts[0]
-                    inv_transforms[site['name']] = base_transform
-                    params[site['name']] = base_transform(transform.inv(site['value']))
+                    inv_transforms[site['name']] = transform.parts[0]
                 else:
                     inv_transforms[site['name']] = transform
-                    params[site['name']] = site['value']
+                params[site['name']] = transform.inv(site['value'])
```

Comment:
this is a bug, which incorrectly defines init params. I added an assertation at `test_beta_bernoulli` in `test_svi` to prevent this regression.